libpod/container_easyjson.go

```diff
@@ -1,4 +1,4 @@
-// +build  seccomp   ostree selinux  varlink exclude_graphdriver_devicemapper
+// +build  seccomp   containers_image_ostree_stub selinux  varlink exclude_graphdriver_devicemapper
```

Comment:
Is this related.

Reply:
Oops - looks like easyjson regenerated. I'll fix.